.eslintrc.json

```diff
@@ -32,11 +32,12 @@
     "no-console": 0,
     "quotes": [
       "error",
-      "single"
+      "single",
+      { "allowTemplateLiterals": true }
```

Comment:
For my education, what does this mean?

Reply:
It's so we can use template literals so these marks without es lint yelling at us ` ` ` `